apps/browser-tests/src/App.tsx

```diff
@@ -105,15 +110,25 @@ function Transactions({
           ],
         });
       } else if (provider === "magiceden") {
-        signPsbtResponse = await magiceden.signPsbt(psbt.toPSBT(), {
```

Comment:
throw an error if `satsConnectWallet` is empty

Reply:
Fixed!